src/mesido/esdl/esdl_heat_model.py

```diff
@@ -2028,8 +2030,8 @@ def equations(x):
 
         q_nominal = self._get_connected_q_nominal(asset)
         density = self.get_density(asset.name, asset.out_ports[0].carrier)
-        mass_flow_max = max_power / eff_max_load / 3600
-        mass_flow_nominal = min(density * q_nominal, mass_flow_max / 2)
+        mass_flow_max_g_per_s = max_power / eff_max_load / 3600 * 1000.0  # Femke check this please
```

Comment:
The factor 1000 is not needed for the unit g/s:
Power [W]
eff [kWh/kg] --> [Wh/g]
eff[Wh/g]*3600[s/h] --> [Ws/g]

Power/(eff *3600) --> g/s

Reply:
Agree. I removed the factor 1000.